opennms-doc/guide-install/src/asciidoc/text/opennms/debian.adoc

```diff
@@ -0,0 +1,152 @@
+* _jicmp6_ and _jicmp_: _Java_ bridge to allow sending _ICMP messages_ from _OpenNMS_ repository.
+* _opennms-core_: _OpenNMS_ core services, e.g. _Provisiond_, _Pollerd_ and _Collectd_ from _OpenNMS_ repository.
+* _opennms-webapp-jetty_: _OpenNMS_ web application from _OpenNMS_ repository
+* _jdk1.8_: _Oracle Java 8_ environment from _OpenNMS_ respository
+* _postgresql_: _PostgreSQL_ database server from distribution repository
+* _postgresql-libs_: _PostgreSQL_ database from distribution repository
+
+With the successful installed packages the _OpenNMS_ platform is installed in the following directory structure:
+
+[source, shell]
+----
+[root@localhost /usr/share/opennms]# tree -L 2
+.
+└── opennms
+   ├── bin
+   ├── data
+   ├── deploy
+   ├── etc -> /etc/opennms
+   ├── instances
+   ├── jetty-webapps
+   ├── lib -> ../java/opennms
+   ├── logs -> /var/log/opennms
+   ├── share -> /var/lib/opennms
+   └── system
+----
+
+[[gi-install-opennms-deb-prepare-pg]]
+==== Prepare PostgreSQL
+
+The _Debian_ package installs also _PostgreSQL_ database and is already initialized and added in the runlevel configuration.
+It is only necessary to start the _PostgreSQL_ database without a restart.
+
+[source, shell]
+----
+service postgresql start
+----
+
+The next step is creating an _opennms_ database user with password and configure the authentication method.
+
+.Create database user _opennms_ and a database _opennms_ with the owner of the created user.
+[source, shell]
+----
+su - postgres
+createuser -P opennms
+createdb -O opennms opennms
+----
+
+NOTE: It is not necessary to change the authentication method in `pg_hba.conf`, it is by default set to `md5` for localhost connections.
+
+.Configure the authentication credentials in the _OpenNMS_ data configuration.
+[source, shell]
+----
+vi ${OPENNMS_HOME}/etc/opennms-datasource.xml
+----
+
+.Add the username and password in *both* sections
+[source, xml]
+----
+<jdbc-data-source name="opennms"
+                    database-name="opennms"
+                    class-name="org.postgresql.Driver"
+                    url="jdbc:postgresql://localhost:5432/opennms"
+                    user-name="** YOUR-OPENNMS-USERNAME **"
+                    password="** YOUR-OPENNMS-PASSWORD **" />
+
+<jdbc-data-source name="opennms-admin"
+                    database-name="template1"
+                    class-name="org.postgresql.Driver"
+                    url="jdbc:postgresql://localhost:5432/template1"
+                    user-name="** YOUR-OPENNMS-USERNAME **"
+                    password="** YOUR-OPENNMS-PASSWORD **" />
+----
+
+[[gi-install-opennms-deb-init]]
+==== Initialize OpenNMS
+
+_OpenNMS_ is now configured to access the database.
+It is required to set the _Java_ environment running _OpenNMS_ and initialize the database schema.
+
+.Set _Java_ environment
+[source, shell]
+----
+${OPENNMS_HOME}/bin/runjava -s
```

Comment:
Just want to get people used to ${OPENNMS_HOME} so we don't have to differentiate all the time in several other places.